upload-api/functions/ucan-invocation-router.js

```diff
@@ -152,72 +155,127 @@ export async function ucanInvocationRouter(request) {
 
   const { UPLOAD_API_DID } = process.env
   const { PRIVATE_KEY, STRIPE_SECRET_KEY } = Config
-  const serviceSigner = getServiceSigner({ did: UPLOAD_API_DID, privateKey: PRIVATE_KEY })
+  const serviceSigner = getServiceSigner({
+    did: UPLOAD_API_DID,
+    privateKey: PRIVATE_KEY,
+  })
 
-  const tasksStorage = createTasksStorage(AWS_REGION, invocationBucketName, workflowBucketName)
-  const receiptsStorage = createReceiptsStorage(AWS_REGION, taskBucketName, invocationBucketName, workflowBucketName)
-  const allocationsStorage = createAllocationsStorage(AWS_REGION, allocationTableName, {
-    endpoint: dbEndpoint,
+  const agentStore = AgentStore.open({
```

Comment:
For past experiences on testing, we end up always allowing to pass the bucket clients so that either Lambdas, or tests give them to the implementation of a Store. I would recommend this pattern to keep consistency in the project, as well as make testing easy then

Reply:
You can actually pass `store.connection.channel` instead of `store.connection.address` to de exactly that.